app/Policies/ReservationPolicy.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace App\Policies;
+
+use App\Models\User;
+use App\Models\reservation;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class ReservationPolicy
+{
+    use HandlesAuthorization;
+
+    /**
+     * Determine whether the user can view any models.
+     *
+     * @param  \App\Models\User  $user
+     * @return \Illuminate\Auth\Access\Response|bool
+     */
+    public function viewAny(User $user)
+    {
+        return $user->role->title === "owner";
```

Comment:
meerdere return statements kan niet, dit was in de andere policies eerst fout gedaan 😅